src/main/kotlin/app/Configurator.kt

```diff
@@ -4,163 +4,116 @@
 package app
 
 import app.utils.Options
-import app.utils.UsernameValidator
-import org.apache.commons.codec.digest.DigestUtils
-import java.io.File
+import app.utils.PasswordHelper
+import com.fasterxml.jackson.core.JsonParseException
+import com.fasterxml.jackson.databind.JsonMappingException
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
+import com.fasterxml.jackson.module.kotlin.KotlinModule
 import java.io.IOException
```

Comment:
But we can also have our own rules as long as we document them. The notion I proposed is used in python and C++ by google. It's convenient because it makes it easy to see standard and custom stuff separated. But then again, maybe your IDE is sorting them all automatically, I don't know.